extensions/vscode/src/utils/guid.ts

```diff
@@ -0,0 +1,9 @@
+// Copyright (C) 2024 by Posit Software, PBC.
+
+export const GuidRegex =
+  /(?:\{{0,1}(?:[0-9a-fA-F]){8}-(?:[0-9a-fA-F]){4}-(?:[0-9a-fA-F]){4}-(?:[0-9a-fA-F]){4}-(?:[0-9a-fA-F]){12}\}{0,1})/;
```

Comment:
Very nice! Played around a bit in `regexr` to double check this to the best of my ability.

Reply:
Thanks!